mgplvm/rdist/rGP.py

```diff
@@ -72,20 +60,20 @@ def __init__(self,
         #self._nu_i = nn.Parameter(data=torch.zeros(1), requires_grad=True)
 
         #initialize covariance parameters
-        _scale = torch.ones(n_samples, self.d, m) * _scale  #n_diag x T
+        _scale = torch.ones(n_samples, self.d, m) * _scale * (
+            1 + torch.randn(m) / 100)  #n_diag x T
```

Comment:
Does it make sense to add the randomness here? Or should we just add a test that specifically tests for it instead? 

If it's useful to break symmetry for training purposes, then I guess fine to add it here, but if it's mostly for testing purposes, better to manually adjust the scale in the initialized test versions.

Reply:
you're probably right, I was just being lazy

Reply:
reverted this, but moved asymmetric scale to test.